test/integration/pack/test_pack.py

```diff
@@ -223,6 +223,40 @@ def prepare_tgz_test_cases(tt_cmd) -> list:
                                                          ".rocks")),
             ]
         },
+        {
+            "bundle_src": "cartridge_app",
+            "cmd": tt_cmd,
+            "pack_type": "tgz",
```

Comment:
A test to check that OnlyValidArgs works fine is needed.

Reply:
Added.